src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,101 @@
+package mate.jdbc.dao;
+
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+import mate.jdbc.util.DataProcessingException;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    @Override
+    public List<Manufacturer> getAll() {
+        List<Manufacturer> getAllManufacturers = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+             Statement getAllInitsStatement = connection.createStatement()) {
+            ResultSet resultSet = getAllInitsStatement.executeQuery("SELECT * FROM manufacturers WHERE is_deleted = false");
+            while (resultSet.next()) {
+                String name = resultSet.getString("name");
+                String country = resultSet.getString("country");
+                Long id = resultSet.getObject("id", Long.class);
+                Manufacturer manufacturer = new Manufacturer();
+                manufacturer.setName(name);
+                manufacturer.setCountry(country);
+                manufacturer.setId(id);
+                getAllManufacturers.add(manufacturer);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get all manufacturers from DB", e);
+        }
+        return getAllManufacturers;
+    }
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String insertManufacturerRequest = "INSERT INTO manufacturers(name, country) values(?, ?);";
```

Comment:
@EliosPride move all sql queries to constants

Reply:
@Garret-vin ++